burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
```

Comment:
I thought extension & filter is applied in each coordinate direction separately, at least in gaussian_filter, so:
start:
```
3 3
4 4
```
extend in x direction:
```
1 3 3 1
1 4 4 1
```
apply in x direction:
```
5/2 5/2
13/4 13/4
```
extend in y direction (I made the numbers up):
```
1 1
5/2 5/2
13/4 13/4
4 4
```
finally apply in y direction.